src/main/resources/templates/db_scripts/schema.json

```diff
@@ -1125,6 +1031,11 @@
       "run": "after",
       "snippetPath": "inventory-hierarchy/correctGetUpdatedInstanceIdsView.sql",
       "fromModuleVersion": "23.1.0"
+    },
+    {
+        "run":"after",
+        "snippetPath":"dropCallNumberNormalizationFunctions.sql",
+        "fromModuleVersion":"24.0.0"
```

Comment:
Can you remove this entry from schema.json:

```
    {
      "run": "before",
      "snippetPath": "createCallNumberNormalizationFunctions.sql",
      "fromModuleVersion": "19.2.0"
    },
```

Can you remove this file:
https://github.com/folio-org/mod-inventory-storage/blob/860fa8e43f132d1a18f449c74318469e11eef701/src/main/resources/templates/db_scripts/createCallNumberNormalizationFunctions.sql

Reply:
I just forgot to push the changes. =